lib/perl/Genome/Model/Tools/Sx/Split/ByNs.t

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env genome-perl
+
+use strict;
+use warnings;
+
+use above 'Genome';
+
+use Genome::Utility::Test;
+use File::Spec;
+use File::Temp;
+use Test::More tests => 5;
+
+my $pkg = 'Genome::Model::Tools::Sx::Split::ByNs';
+use_ok($pkg);
+
+my $dir = File::Spec->join( Genome::Config::get('test_inputs'), 'Genome-Model-Tools-Sx', 'SplitByNs');
+my $in_fasta = File::Spec->join($dir, 'in.fasta');
+ok(-s $in_fasta, 'in fasta');
+my $expected_fasta = File::Spec->join($dir, 'expected.fasta');
+ok(-s $expected_fasta, 'expected fasta');
+
+my $tmp_dir = File::Temp::tempdir(CLEANUP => 1);
```

Comment:
This could be a `Genome::Sys->create_temp_directory` so LSF cleans it up even if things go south.